.github/workflows/release.yaml

```diff
@@ -1,88 +1,34 @@
 name: Release Operator
 
 on:
-  workflow_dispatch:
```

Comment:
I strongly believe this is should **not** be removed. The new release process that is proposed adds many extra steps, both local and remote, reducing the profile of people that can release the operator and even if it adds some validation step to avoid human error, it doesn't guarantee mistakes won't be made. The release button could be enhance to have an intermediate step that could add some extra validation.

Reply:
Far I can add that back in.

Reply:
Will add it in my following PR